lib/autoyast.rb

```diff
@@ -101,6 +100,12 @@ def profile
     builder.to_xml
   end
 
+  def outgoing_ip
+    output = Cheetah.run("ip", "route", "ls", stdout: :capture)
```

Comment:
There is a straight forward way of getting the outgoing ip address: 
`ip r get 1 | awk '{print $NF; exit}' `
or if you don't want to use `awk`
`ip r get 1` and use the last word of the first line only.